blocks/identity-block/features/account-management/default.jsx

```diff
@@ -0,0 +1,169 @@
+import React, { useEffect, useState } from "react";
+import PropTypes from "@arc-fusion/prop-types";
+import { useFusionContext } from "fusion:context";
+import getProperties from "fusion:properties";
+import getTranslatedPhrases from "fusion:intl";
+import { Heading } from "@wpmedia/arc-themes-components";
+import useIdentity from "../../components/identity";
+import { GoogleSignInProvider } from "../../components/social-sign-on/utils/googleContext";
+import EmailEditableFieldContainer from "./_children/EmailEditableFieldContainer";
+import PasswordEditableFieldContainer from "./_children/PasswordEditableFieldContainer";
+import SocialEditableSection from "./_children/SocialEditableSection";
+
+const BLOCK_CLASS_NAME = "b-account-management";
+
+export function AccountManagementPresentational({ header, children }) {
+	return (
+		<div className={`${BLOCK_CLASS_NAME}__section`}>
+			<Heading>{header}</Heading>
+			{children}
+		</div>
+	);
+}
+
+function AccountManagement({ customFields }) {
+	const [loggedIn, setLoggedIn] = useState(false);
+	const [isLoading, setIsLoading] = useState(true);
+	const [email, setEmail] = useState("");
+	const [hasPassword, setHasPassword] = useState();
+	const [hasGoogle, setHasGoogle] = useState(false);
+	const [hasFacebook, setHasFacebook] = useState(false);
+
+	const { redirectURL, showEmail, showPassword, showSocialProfile } = customFields;
+
+	// get properties from context for using translations in intl.json
+	// See document for more info https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/2538275032/Lokalise+and+Theme+Blocks
+	const { arcSite, isAdmin } = useFusionContext();
+	const { locale } = getProperties(arcSite);
+	const phrases = getTranslatedPhrases(locale);
+
+	const { isInitialized, Identity } = useIdentity();
+
+	// istanbul ignore next
+	useEffect(() => {
+		const checkLoggedInStatus = () =>
+			Identity.isLoggedIn().then((isLoggedIn) => {
+				if (!isLoggedIn) {
+					window.location = redirectURL;
+					return;
+				}
+				setLoggedIn(true);
+			});
+		if (Identity && !isAdmin) {
+			checkLoggedInStatus();
+		}
+	}, [Identity, isAdmin, redirectURL]);
+
+	// istanbul ignore next
+	useEffect(() => {
+		const getProfile = () =>
+			Identity.getUserProfile().then((profileObject) => {
+				const { email: loggedInEmail, identities } = profileObject;
+
+				if (loggedInEmail) {
+					setEmail(loggedInEmail);
+				}
+
+				const passwordProfile = identities.filter(
+					({ type }) => type === "Password" || type === "Identity"
+				);
+
+				setHasPassword(passwordProfile?.length > 0);
+				setHasGoogle(identities.filter(({ type }) => type === "Google").length > 0);
+				setHasFacebook(identities.filter(({ type }) => type === "Facebook").length > 0);
+				// todo: in future ticket, handle errors
+				// else {
+				//   setError('No email found');
+				// }
```

Comment:
We should remove these.  If you want to bring them back later, create a story and link this comment/commit.